core/model/src/main/java/org/eclipse/sw360/antenna/model/license/LicenseStatement.java

```diff
@@ -10,10 +10,7 @@
  */
 package org.eclipse.sw360.antenna.model.license;
 
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.List;
-import java.util.Objects;
+import java.util.*;
```

Comment:
Nit: The commit message just repeats in plain word what's obvious from the diff, but lacks context *why* this change makes sense.